src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.spec.ts

```diff
@@ -110,5 +138,26 @@ describe('DotEditPageNavComponent', () => {
                 expect(item.nativeElement.textContent).toContain(labels[index]);
             });
         });
+
+        describe('license', () => {
+            beforeEach(() => {
+                component.pageState = new DotRenderedPageState(mockUser, mockDotRenderedPage);
+                component.enterpriselicense = false;
+                fixture.detectChanges();
+            });
+
+            it('should have layout option disabled because user does not has a proper license', () => {
+                const menuListItems = fixture.debugElement.queryAll(By.css('.edit-page-nav__item'));
+                expect(menuListItems[1].nativeElement.classList).toContain('edit-page-nav__item--disabled');
+            });
+
+            it('should the layout option have the proper attribute & message key for tooltip', () => {
+                const menuListItems = fixture.debugElement.queryAll(By.css('.edit-page-nav__item'));
+                const layoutTooltipTextAttr = menuListItems[1].nativeElement.attributes['ng-reflect-text'].value;
```

Comment:
I "think" you can do ```menuListItems[1].componentInstance``` and check the tooltip attr there, this ```ng-reflect-text``` doesn't look safe.